editor/src/core/workers/parser-printer/parser-printer.test-utils.ts

```diff
@@ -77,12 +77,14 @@ import { getUtopiaIDFromJSXElement } from '../../shared/uid-utils'
 import { fastForEach } from '../../shared/utils'
 import { addUniquely, flatMapArray } from '../../shared/array-utils'
 import { optionalMap } from '../../shared/optional-utils'
+import { emptyComments, parsedComments, ParsedComments } from './parser-printer-comments'
 
 export const singleLineCommentArbitrary: Arbitrary<SingleLineComment> = lowercaseStringArbitrary().map(
   (text) => {
     return {
       type: 'SINGLE_LINE_COMMENT',
       comment: text,
+      rawText: text,
```

Comment:
This looks a little suspicious, `comment` would from what I can tell never be equal to `rawText`.